components/core/CMakeLists.txt

```diff
@@ -186,13 +186,17 @@ set(SOURCE_FILES_unitTest
         src/clp/clp/run.hpp
         src/clp/clp/utils.cpp
         src/clp/clp/utils.hpp
+        src/clp/database_utils.cpp
+        src/clp/database_utils.hpp
         src/clp/dictionary_utils.cpp
         src/clp/dictionary_utils.hpp
         src/clp/DictionaryEntry.hpp
         src/clp/DictionaryReader.hpp
         src/clp/DictionaryWriter.hpp
         src/clp/EncodedVariableInterpreter.cpp
         src/clp/EncodedVariableInterpreter.hpp
+        src/clp/FileReader.cpp
```

Comment:
Why do we move FileReader to `src/clp`, but keep BufferReader and FileWriter in `src/`?